src/scss/partials/_header.scss

```diff
@@ -15,6 +19,8 @@ $header--transparent-opacity        : 0.7;
 // Menulog theme overrides
 @if ($theme == 'ml') {
     $header-buttonIcon-color: $green;
+    $header-buttonIcon-color: $green;
```

Comment:
This variable is already present on the line above, so can delete this one.

Reply:
Oops! I missed this one..